src/dependency.json

```diff
@@ -0,0 +1,8 @@
+{
+  "charm": {
```

Comment:
same as here https://github.com/canonical/postgresql-k8s-operator/pull/199#discussion_r1275760205

Reply:
My bad. I uploaded this update to another [branch](https://github.com/canonical/postgresql-operator/pull/191/commits/6ad056268d7c845e5063189f3d77f55c2aa0d48e) last week. I updated it in this branch/PR on https://github.com/canonical/postgresql-operator/pull/187/commits/735b65412a97b432cd6eaa77768069023ad8cd4f.